Not actually need that patch, just for compatibility with previous versions
+++ ./src/Config.template.h
@@ -62,9 +62,9 @@
 
 constexpr char SCHEME[]         = "https://";
 constexpr char STATICSCHEME[]   = "https://";
-constexpr char LOCAL_SAVE_DIR[] = "Saves";
+constexpr char LOCAL_SAVE_DIR[] = "saves";
 constexpr char STAMPS_DIR[]     = "stamps";
-constexpr char BRUSH_DIR[]      = "Brushes";
+constexpr char BRUSH_DIR[]      = "brushes";
 
 constexpr int httpMaxConcurrentStreams = 50;
 constexpr int httpConnectTimeoutS      = 15;
